src/5e-SRD-Weapon-Properties.json

```diff
@@ -1,88 +1,90 @@
 [
   {
     "index": "ammunition",
-    "name": "Ammunition",
+    "name": "ammunitionName",
     "desc": [
-      "You can use a weapon that has the ammunition property to make a ranged attack only if you have ammunition to fire from the weapon. Each time you attack with the weapon, you expend one piece of ammunition. Drawing the ammunition from a quiver, case, or other container is part of the attack (you need a free hand to load a one-handed weapon).",
-      "At the end of the battle, you can recover half your expended ammunition by taking a minute to search the battlefield. If you use a weapon that has the ammunition property to make a melee attack, you treat the weapon as an improvised weapon (see \"Improvised Weapons\" later in the section). A sling must be loaded to deal any damage when used in this way."
+      "ammunitionDesc1",
+      "ammunitionDesc2"
```

Comment:
Would it make more sense for `ammunitionDesc` to be a list instead?

Reply:
Yes and No. That's one of my concerns about this approach instead of just mimicking the whole file structure and start translating it. We'll talk more today.